style.css

```diff
@@ -0,0 +1,504 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.flex {
+  display: flex;
+}
+
+.grid {
+  display: grid;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+nav {
+  position: fixed;
+  width: 100%;
+  background-color: white;
+  z-index: 1;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  height: auto;
+}
+
+.float-left {
+  width: 100%;
+}
+
+.nav-1 {
+  justify-content: space-between;
+  height: 30px;
+  align-items: center;
+  padding-left: 2%;
+  padding-right: 2%;
+  background-color: #131313;
+}
+
+.nav-1 a {
+  color: grey;
+  padding-right: 8px;
+  font-size: 2vh;
+}
+
+.ftr-links a {
+  margin-left: 5%;
+  margin-right: 5%;
+  color: #e1e1e1;
+}
+
+.lists a {
+  color: gray;
+}
+
+.section-3-top a {
+  padding-left: 1vw;
+  padding-right: 1vw;
+  color: #f42;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your ` color: #f42;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```CSS
.color-orange{
   color: #f42;
}
```
**Html file**

```HTML
<div class="color-orange">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_